### PR TITLE
Jd/dpc 3692 client tokens

### PR DESCRIPTION
**Pull request by jdettmannnava**
## 🎫 Ticket

https://jira.cms.gov/browse/DPC-3692

## 🛠 Changes

- Can create and delete client tokens
- Form components
- Client Token new and show components
- Table row accepts delete

## ✅ Acceptance Validation

Best if run locally and can match to spec in ticket

delete modal
new token
new token error
show token

## 🔒 Security Implications

- [ ] This PR adds a new software dependency or dependencies.
- [ ] This PR modifies or invalidates one or more of our security controls.
- [ ] This PR stores or transmits data that was not stored or transmitted before.
- [ ] This PR requires additional review of its security implications for other reasons.

If any security implications apply, add Jason Ashbaugh (GitHub username: StewGoin) as a reviewer and do not merge this PR without his approval.

### CODE REVIEW

File: dpc-portal/app/controllers/client_tokens_controller.rb [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+# Hanles client token requests
+class ClientTokensController < ApplicationController
+  before_action :load_organization
+
+  def new
+    render Page::ClientToken::NewTokenComponent.new(@organization)
```
Comment (chris-ronning-ny) [inline dpc-portal/app/controllers/client_tokens_controller.rb:8]: Is there anything preventing this or line 15 from being called before `load_organization` sets this instance variable?

Comment (jdettmannnava) [reply dpc-portal/app/controllers/client_tokens_controller.rb:8]: before_action calls load_organization before anything happens, so I don't see how.

**Review by chris-ronning-ny — APPROVED** [review]
I am very much not a Rails expert, but from what I can tell, this looks good to me.

File: dpc-portal/app/components/core/form/text_area_component.html.erb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+        <div class="margin-bottom-4">
+          <%= label_tag attribute, label, class: "usa-label" %>
```
Comment (MEspositoE14s) [inline dpc-portal/app/components/core/form/text_area_component.html.erb:2]: I didn't know there were helpers for tab elements like this!

**Review by MEspositoE14s — APPROVED** [review]
I'm in no position to judge the fine details of the code, but broadly it all makes sense, and when I run it locally the components look good in LookBook and all the tests pass.  👍

**Pull request merged**